### DISCUSSION TITLE
benchmark/

### DISCUSSION THREAD

**Discussion by giscus** [jiegec/blog-source#12] [General]
# benchmark/

杰哥的{运维，编程，调板子}小笔记

https://jia.je/benchmark/

**Comment by MrLinWang** [comment]
请问您测试出来的数据是使用的哪个版本的编译器？

**Reply by MrLinWang** [reply]
哦，看到了 GCC 12.2.0 from Debian Bookworm。

**Reply by jiegec** [reply]
对，大部分是用的 GCC 12.2.0，LoongArch 是用的 GCC 14.2.0，我把这个信息添加到开头了。

**Comment by prwang** [comment]
阿里云上有zen 5c了 (ecs.g9ae、抢占式实例）
https://browser.geekbench.com/v6/cpu/15048870

**Reply by jiegec** [reply]
性能已测完并更新。
